src/main/java/com/vinylteam/vinyl/security/LogoutHandlerService.java

```diff
@@ -0,0 +1,34 @@
+package com.vinylteam.vinyl.security;
+
+import com.vinylteam.vinyl.service.JwtService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.web.authentication.logout.LogoutHandler;
+import org.springframework.stereotype.Service;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+@Service
+public class LogoutHandlerService implements LogoutHandler {
+
+    @Autowired
+    private JwtService jwtService;
+
+    @Autowired
+    private InMemoryLogoutTokenStorageService logoutStorageService;
+
+    @Override
+    public void logout(HttpServletRequest request, HttpServletResponse response, Authentication authentication) {
+
+        //extract token
```

Comment:
Please remove these comments and never use them
Your code should be self descriptive